db/migrate/20160718020648_create_focus_areas.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFocusAreas < ActiveRecord::Migration
+  def change
+    create_table :focus_areas do |t|
```

Comment:
@NickyBobby They are something separate. Focus areas can be associated with activities, but activities do not have to have a focus area.